src/main/java/com/ParqueCore/ParkBeto/enums/FuncionarioStatus.java

```diff
@@ -0,0 +1,6 @@
+package com.ParqueCore.ParkBeto.enums;
+
+public enum FuncionarioStatus {
+	ATIVO,
+	INATIVO
+}
```

Comment:
Faça uma alteração nesse enum pf

public enum FuncionarioStatus {
    ATIVO("Ativo"),
    INATIVO("Inativo");

    private final String status;

    FuncionarioStatus(String status) {
        this.status = status;
    }

    public String getStatus() {
        return status;
    }
}

Reply:
Faz isso pros outros enums também pf